lib/workers/repository/validate.js

```diff
@@ -0,0 +1,92 @@
+const { migrateAndValidate } = require('../../config/migrate-validate');
+
+async function getRenovatePrs(branchPrefix) {
+  return (await platform.getPrList())
+    .filter(pr => pr.state === 'open')
+    .filter(pr => pr.branchName && !pr.branchName.startsWith(branchPrefix))
+    .filter(pr => pr.title && pr.title.match(/renovate/i));
```

Comment:
@rarkins I'm not sure that this restriction makes sense. I was about to suggest this feature but I found it when looking through PRs. 

If it's not a renovate PR but it _has_ an updated `renovate.json` file then shouldn't that be enough?